lesson_02/script.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+//<li>
+//<a href="1.html"><img src="bookmark.png" alt="ブックマーク" />これです</a>
+//</li>
+
+const li = document.createElement("li"); //<li></li>
+
+const a = document.createElement("a"); //<a href="1.html"></a>
+a.text = "これです";
+a.href = "1.html";
+
```

Comment:
The code is
For each variable definition
For each instruction to the element
For each DOM operation
I think it will be easier to read if you write it separately.